packages/react-menu/src/components/Menu/useMenuPopup.tsx

```diff
@@ -76,14 +89,21 @@ export const useMenuPopup = (state: UseMenuPopupState) => {
       originalProps?.onClick?.(e);
     };
 
-    return React.createElement(
+    const popupContent = React.createElement(
       Component as React.ElementType,
       {
         ...newProps,
         ref: menuPopupRef,
       },
       menuList,
     );
+
+    // Only outermost menu in a portal, other nested menus should be rendered in the same portal
```

Comment:
Can you please also add "why nested menus should be rendered in the same portal"?